substreams/crates/tycho-substreams/src/entrypoint.rs

```diff
@@ -0,0 +1,48 @@
+use crate::models::{entry_point_params::TraceData, EntryPoint, EntryPointParams};
+
+fn get_entrypoint_id(target: &Vec<u8>, signature: &String) -> String {
+    let target = hex::encode(target);
+    format!("{target}:{signature}")
+}
+
+/// Creates an entrypoint and its parameters.
+pub fn create_entrypoint(
+    target: Vec<u8>,
+    signature: String,
+    component_id: String,
+    trace_data: Option<TraceData>,
```

Comment:
@louise-poole I know you said this shouldn't be optional.. but that is the definition of `oneof` 😕

Reply:
aaaaahhhh ok ok, that wasn't clear to me (that we wanted to that here)